babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -236,6 +236,39 @@ SqlCreateAttribute CreateTableAttributeJournalTable() :
     { return new SqlCreateAttributeJournalTable(id, getPos()); }
 }
 
+SqlCreateAttribute CreateTableAttributeJournal() :
+{
+  JournalType journalType = JournalType.UNSPECIFIED;
```

Comment:
Since this function contains fairly complex parsing logic, suggest not initializing these up here and instead explicitly setting the values to UNSPECIFIED below.

That would make it easier for the reader to easily identify in which cases UNSPECIFIED is being set.